framework/src/base/NonlinearSystemBase.C

```diff
@@ -145,6 +143,13 @@ NonlinearSystemBase::NonlinearSystemBase(FEProblemBase & fe_problem,
     _has_nodalbc_save_in(false),
     _has_nodalbc_diag_save_in(false)
 {
+  _Re_non_time_tag = _fe_problem.addVectorTag("NONTIME");
+  _Re_non_time = &addVector(_Re_non_time_tag, false, GHOSTED);
+
+  // Don't need to add the matrix - it already exists (for now)
+  _Ke_non_time_tag = _fe_problem.addMatrixTag("NONTIME");
```

Comment:
Maybe assign this tag to the system matrix in `_tagged_matrices `?

Reply:
good point